asab/api/service.py

```diff
@@ -24,6 +21,31 @@ def __init__(self, app, service_name="asab.ApiService"):
 		super().__init__(app, service_name)
 		self.WebContainer = None
 		self.ZkContainer = None
+		self.AttentionRequired = {}
+
+	def attention_required(self, attention_key = None):
+
+		if attention_key is None:
+			# update the list with attention field
+			attention_key = str(uuid.uuid4())
+			self.AttentionRequired.update({attention_key: 1})
+
+		if self.ZkContainer is not None:
+			self.ZkContainer.advertise(
+				data=self._build_zookeeper_adv_data(),
+				path="/run/{}.".format(self.App.__class__.__name__),
+			)
+		return attention_key
+
+	def remove_attention(self,attention_key):
+		if attention_key is not None:
```

Comment:
@ateska I am not sure if this condition is required.

Reply:
No it is not needed. `KeyError` exception is fine when `attention_key` is invalid/unknown.